packages/conf-bpftool/conf-bpftool.0.1.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "Virtual package for system installation of bpftool"
+maintainer: ["Lee Koon Wen"]
+authors: ["Lee Koon Wen"]
+license: ["ISC" "BSD-3-Clause"]
+homepage: "https://github.com/koonwen/ocaml-libbpf"
+doc: "https://koonwen.github.io/ocaml-libbpf"
+bug-reports: "https://github.com/koonwen/ocaml-libbpf/issues"
+depends: [
+  "dune" {>= "3.13"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/koonwen/ocaml-libbpf.git"
+available: [ os = "linux" ]
+
+depexts: [
+  [ "linux-tools-common" ] {os-distribution = "ubuntu"}
+  [ "bpftool" ]            {os-distribution = "debian"}
+  [ "bpftool" ]		   {os-distribution = "fedora"}
+]
+url {
+  src:
+    "https://github.com/koonwen/ocaml-libbpf/releases/download/v0.1.0/libbpf-0.1.0.tbz"
+  checksum: [
+    "sha256=c0b543a2628f4c27d2eaaefb1b8c0cc81ef56a2437de44e540770a9a689ab1fe"
+    "sha512=1ff0b508123c131df16cf53ca43681a9d5b2120ffbb7409e264be11ca37ae74057acc7121d9811fb8eeb5ab224a71d11af50498e28fe367d6a21d29011a937e5"
+  ]
+}
```

Comment:
```suggestion
available: [ os = "linux" ]
depexts: [
  [ "linux-tools-common" ] {os-distribution = "ubuntu"}
  [ "bpftool" ]            {os-distribution = "debian"}
  [ "bpftool" ]		   {os-distribution = "fedora"}
]
flags: conf
```

conf packages should not contain any sources, exceptions can be made for small scripts to check if they libraries are properly installed